app/src/main/java/com/quran/labs/androidquran/data/AyahInfoDatabaseProvider.java

```diff
@@ -31,4 +32,8 @@ public AyahInfoDatabaseHandler getAyahInfoHandler() {
     }
     return databaseHandler;
   }
+
+  public int getPageWidth() {
+    return Integer.valueOf(widthParameter.substring(1));
```

Comment:
this is a bit of a hack, but it's the easiest way to do this at the moment. in the future, should just provide the actual width.